pkg/kubelet/volumemanager/cache/desired_state_of_world.go

```diff
@@ -412,3 +430,36 @@ func (dsw *desiredStateOfWorld) isDeviceMountableVolume(volumeSpec *volume.Spec)
 
 	return false
 }
+
+func (dsw *desiredStateOfWorld) AddErrorToPod(podName types.UniquePodName, err string) {
```

Comment:
how big can this get?  is it possible for this to grow unbounded ?

Reply:
Good question, errors are stored in `sets.String`, so it *can* grow if every error has a different text. I added a limit of 10, just in case there is a bug in the populator.